### DISCUSSION TITLE
Lack of documentation

### DISCUSSION THREAD

**Discussion by TitaniumHocker** [bczsalba/pytermgui#97] [Q&A] [answered]
Current documentation seems to be just autogenerated API documentation, without something like quickstart, application guidelines, style system(tim and especially aliases) guides, etc.
Are there any plans to create detailed documentation with the sections I described?

For me poor documentation is the main reason not to use pytermgui in my app. But widget system and styling seems quite interesting.

**Comment by bczsalba** [comment, marked as answer]
Hey there! There _is_ an in-development documentation that is way better and has a couple of detailed guides, but I unfortunately got wrapped up in university work (specifically a game development minor) and haven't quite had a chance to finish it up.

I'll work on it a bit more in the coming days since I'm ahead of schedule in uni, and I hope to release it sometime soon. Until then, you can run `mkdocs serve` in the new-docs branch to see it in action.

Thanks for the interest!

**Reply by TitaniumHocker** [reply]
Thx for `mkdocs serve` hint. Good luck with your university work!

**Reply by jared201** [reply]
agree, some of the getting started examples don't run on the fly, need some tweaking though.

**Reply by bczsalba** [reply]
> agree, some of the getting started examples don't run on the fly, need some tweaking though.

Are you talking about the current live docs, or the ones in [development](https://ptg-docs-beta.netlify.app/)?
